### ISSUE TITLE
Not ready to install R package

### ISSUE THREAD

**Issue by AngelCampos** [Zhangxf-ccnu/scTSSR#1]
The code in this repository would be more valuable if is ready to install using {devtools}.

Please consider updating the structure of the repository to make your package installable from within R.
The ["R packages"](https://r-pkgs.org/git.html) book by Hadley Wickham explains the process in detail. 

An example of a "correctly" structured repository can be found [here](https://github.com/tidyverse/dplyr).
Basically, you just need to extract the content of your "pkg" folder to the top-level folder, and eliminate *Rproj and *.Rhistory files.

Looking forward to testing your package.
Best

**Comment by Zhangxf-ccnu** [comment]
Hi AngelCampos,

      Thank you for your good suggestions. We have updated the structure of the repository. 

Regards,

xiaofei
